common/src/main/java/com/wynntils/wynn/model/map/poi/CombatPoi.java

```diff
@@ -24,6 +24,9 @@ public Texture getIcon() {
 
     @Override
     public float getMinZoomForRender() {
+        if (kind == CombatKind.CAVES) {
```

Comment:
Eventually all types should be configurable

Reply:
I agree. Doing this in a way that makes it useful will be a bit of a challenge. I really disliked the system in Legacy. I want some kind of hierarchical system, were "subtypes" inherts values of supertypes, unless overridden. So you can like set a value for all "combat POI" and then make an exception for caves, etc. Or make most services appear only when really zoomed in, except for identifiers which show up "earlier", etc.